content/release-notes/2023-04-11-console.md

```diff
@@ -0,0 +1,23 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- API: Added `consumption_period_start` and `consumption_period_end` properties to the [Get project details](https://api-docs.neon.tech/reference/getproject) API. These properties provide the date and time when Neon starts and stops measuring usage for the current billing period.
+- Control Plane: Added support for the [pg_stat_statements](https://www.postgresql.org/docs/current/pgstatstatements.html) extension, which provides tracking of plan and execution statistics for SQL statements. For more information about PostgreSQL extensions supported by Neon, see [PostgreSQL extensions](/docs/extensions/pg-extensions/).
+- Control Plane: Added an option to allow the Neon Control Plane to provide a compute node configuration specification in an API call after a compute node is started.
+- Integrations: Added information to the Neon Vercel Integration UI to indicate that Neon securely stores passwords. Neon stores a database user's password in a secure storage vault associated with the Neon project.  
+- UI: You can now configure the number of seconds of inactivity after which a compute is automatically suspended. This feature is referred to as _Configurable auto-suspend compute_. For example, you can increase the number of seconds to reduce how often a compute is suspended, or you can disable _Auto-suspend_ entirely to ensure that a compute remains active. The maximum setting is 86400 seconds (24-hours). A setting of 0 means use the default (5 minutes / 300 seconds), and a setting of -1 means never suspend the compute.
+- UI: Added a **Storage** widget to the Neon **Dashboard** that shows current project storage size, number of branches, and data size for the primary branch. These metrics previously appeared in the **Usage** widget.
+- UI: Project limits for shared projects are now checked against the project owner's limits. Previously, limits were checked against the project user's limits, which could prevent a Free Tier user from making full use of a project shared by a Pro plan user.
```

Comment:
It looks more like `Bug fix` for API (Console)

Reply:
Thank you. Relabeled and moved to bug fixes section.